src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIModelMapper.ChatCompletion.cs

```diff
@@ -398,28 +398,16 @@ private static ChatTool ToOpenAIChatTool(AIFunction aiFunction)
             strictObj is bool strictValue ?
             strictValue : null;
 
-        BinaryData resultParameters = OpenAIChatToolJson.ZeroFunctionParametersSchema;
-
-        var parameters = aiFunction.Metadata.Parameters;
-        if (parameters is { Count: > 0 })
+        BinaryData functionParameters = OpenAIChatToolJson.ZeroFunctionParametersSchema;
+        if (aiFunction.Metadata.Schema is { } schema)
```

Comment:
Is this still the right check? Will ZeroFunctionParametersSchema ever be used now?